src/components/notification-tray/readme.md

```diff
@@ -0,0 +1,138 @@
+# oui-notification-tray
+
+Notification tray
+
+### Light Theme
+
+![oui-notification-tray light](../../../static/images/screenshot01.png)
+
+### Dark Theme
+
+![oui-notification-tray dark](../../../static/images/screenshot02.png)
+
+### States
+
+![oui-notification-tray states](../../../static/images/screenshot03.png)
+
+![oui-notification-tray states](../../../static/images/animation01.gif)
+
+## Notification Object
+
+```js
+interface NotificationProps {
+  title: string
+  link?: string
+  type: "link" | "info"
+  detail?: string
+  valence: "success" | "fail",
+  read?: boolean
+}
+```
+
+## Usage
+
+```html
+
+  <div style="width: 98%; margin: 4em auto; text-align: center;">
+    <oui-tm-switch></oui-tm-switch>
+    <oui-nav-bar>
+      <div style="flex: 1;"></div>
+      <oui-notification-tray>
+        <oui-notification-drawer></oui-notification-drawer>
+      </oui-notification-tray>
+    </oui-nav-bar>
+  </div>
+  
+  <script>
+    const notifications = [
+      { name: 'Ubuntu Release', type: 'link', detail: 'Ubuntu 19.10 has been released', valence: 'success', read: false },
+      { name: 'Apple Release', type: 'info', detail: 'OSX Catalina has been released', valence: 'fail', read: false }
+    ]
+
+    console.log(this.event)
+
+    const dismiss = (evt) => {
+      alert(`Notification ${evt.detail} has been dismissed`)
+      if (tray.count > 0) {
+        tray.count--
+        console.log(tray.count)
+      }
+    }
+
+    const dismissAll = (evt) => {
+      alert('All notifications have been dismissed')
+      if (tray.unread > 0) {
+        tray.unread = 0
+        console.log(tray.unread)
+      }
+    }
+
+    const drawer = document.querySelector("oui-notification-drawer")
+
+    const tray = document.querySelector("oui-notification-tray")
+    tray.count = notifications.length
+    tray.unread = notifications.length
+
+    const notificationsMapper = () => {
```

Comment:
I think if we could, lets move this style of generation of the elements to a note in the readme (separate from an example), and have the usage example be just the markup

```html
<div style="width: 98%; margin: 4em auto; text-align: center;">
    <oui-tm-switch></oui-tm-switch>
    <oui-nav-bar>
      <div style="flex: 1;"></div>
      <oui-notification-tray>
        <oui-notification-drawer>
          <oui-notification-item name="Ubuntu Release" type="link" detail="Ubuntu 19.10 has been released" valence="success" />
          <oui-notification-item name="Apple Release" type="info" detail="OSX Catalina has been released" valence="fail" />        </oui-notification-drawer>
      </oui-notification-tray>
    </oui-nav-bar>
  </div>
```